travel-event-host-app/src/components/searchSection/types.ts

```diff
@@ -0,0 +1,4 @@
+type CategoriesType = {
+  [key: string]: boolean;
```

Comment:
You could strongly type this with something like:
```suggestion
[key in Category]?: boolean
```

Reply:
no, again we are saving string with spaces values no CategoryType